web/locales/en/plugin__distributed-tracing-console-plugin.json

```diff
@@ -7,20 +7,27 @@
   "Last 12 hours": "Last 12 hours",
   "Last 1 day": "Last 1 day",
   "Last 7 days": "Last 7 days",
+  "Time Range": "Time Range",
   "Select a Time Range": "Select a Time Range",
   "No Tempo instance selected": "No Tempo instance selected",
   "Select a Tempo instance to explore data": "Select a Tempo instance to explore data",
-  "No results found": "No results found",
-  "Clear all filters and try again": "Clear all filters and try again",
-  "Clear all filters": "Clear all filters",
+  "Tempo Instance": "Tempo Instance",
   "Select a Tempo instance": "Select a Tempo instance",
+  "TempoStack and TempoMonolithic instances with multi-tenancy are supported": {
```

Comment:
Should we add this as 2 different strings, not sure if this format is properly supported